management/example_test.go

```diff
@@ -1,181 +0,0 @@
-package management_test
```

Comment:
Removing the examples as they don't provide any usefulness inside the godoc pages as they're not runnable without the proper credentials so they won't display any output: https://pkg.go.dev/github.com/auth0/go-auth0@v0.14.0/management#pkg-examples. 

For more info:
- https://go.dev/blog/examples